Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2734 @@
+            method='POST',
+            endpoint='/atpapi/v2/commands',
+            params={},
+            json_data=payload
+        ).json()
+
+    def get_status_endpoint(self, command_id: str, payload: dict) -> dict[str, Any]:
+        """
+        Client method for get endpoint status
+        Args:
+            command_id (str): Endpoint command_id
+            payload: request body
+
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='POST',
+            endpoint=f'/atpapi/v2/commands/{command_id}',
+            params={},
+            json_data=payload).json()
+
+    def get_sandbox_verdict_for_file(self, sha2: str) -> dict[str, Any]:
+        """
+        Client method for get sandbox verdict for specific file sha256
+        Args:
+            sha2 (str): File SHA256
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='GET',
+            endpoint=f'/atpapi/v2/sandbox/results/{sha2}/verdict',
+            params={},
+            json_data={}).json()
+
+    def get_file_entity(self, sha2: str) -> dict[str, Any]:
+        """
+        Client method for Get File Entity for specific SHA2
+        Args:
+            sha2 (str): File SHA256
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='GET',
+            endpoint=f'/atpapi/v2/entities/files/{sha2}',
+            params={},
+            json_data={}).json()
+
+    def get_sandbox_status(self, command_id: str) -> dict[str, Any]:
+        """
+        Client method to Get Sanbox Status for specific file
+        Args:
+            command_id (str): sandbox command ID
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='GET',
+            endpoint=f'/atpapi/v2/sandbox/commands/{command_id}',
+            params={},
+            json_data={}).json()
+
+    def submit_file_to_sandbox_analyze(self, payload: dict) -> dict[str, Any]:
+        """
+        Client method for sandbox analyze
+        Args:
+            payload: request body
+
+        Returns:
+            return response json
+        """
+        return self.http_request(
+            method='POST',
+            endpoint='/atpapi/v2/sandbox/commands',
+            params={},
+            json_data=payload
+        ).json()
+
+
+''' HELPER FUNCTIONS '''
+
+
+def convert_to_iso8601(timestamp: str) -> str:
+    """ Convert timestamp from iso 8601 format
+
+    Args:
+        timestamp: Any valid timestamp or provide timedelta e.g. now, "<n> days", "<n> weeks",
+        "<n> months", "1 months ago"
+
+    Returns: return timestamp in iso 8601 format.
+
+    """
+    if datetime_from_timestamp := dateparser.parse(timestamp, settings={'TIMEZONE': 'UTC'}):
+        return f'{datetime_from_timestamp.strftime(ISO8601_DATE_FORMAT)[:23]}Z'
```

Comment:
is the extra `Z` here necessary, given there's one in the time format constant?

Reply:
As discussed, I have modified the date format constant and replace %f with %L

Reply:
Kindly check these changes in the latest revision. I have tried with %L in the date format constant but it will not produce any output. Therefore, stick with my old solution %f and then trim the 3 digits.